samples/apps/copilot-chat-app/WebApp/src/components/chat/typing-indicator/TypingIndicator.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import { Image, makeStyles } from '@fluentui/react-components';
+import React from 'react';
+import typingBalls from '../../../assets/typing-balls-light.svg';
+//const typingBalls = require('../../../assets/typing-balls-light.svg');
```

Comment:
nit: dead code? :)

Reply:
yes... It is indeed a dead code. will remove it.